docs/docs/recipes/transforming-data.md

```diff
@@ -54,3 +54,66 @@ export const query = graphql`
 
 - [Tutorial on transforming Markdown to HTML](/tutorial/part-six/#transformer-plugins) using `gatsby-transformer-remark`
 - Browse available transformer plugins in the [Gatsby plugin library](/plugins/?=transformer)
+
+## Transforming images into grayscale using GraphQL
+
+### Prerequisites
+
+- A [Gatsby site](/docs/quick-start) with `gatsby-config.js` and an `index.js` page
+- The `gatsby-image`, `gatsby-transformer-sharp`, and `gatsby-plugin-sharp` packages installed and added to the plugins array in `gatsby-config`
+- A source plugin installed, such as `gatsby-source-filesystem`
+- An image (`.jpg`, `.png`, `.gif`, `.svg`, etc.) in the `src/images` folder
+
+### Directions
+
+1. Make sure that the config file has the correct information.
+
+Edit your gatsby-config.js file to source images and configure plugins for Gatsby's GraphQL data layer. A common approach is to source them from an images directory using the gatsby-source-filesystem plugin:
+
+```JavaScript:title=gatsby-config.js
+
+ plugins: [
+   {
+     resolve: `gatsby-source-filesystem`,
+     options: {
+       name: `images`,
+       path: `${__dirname}/src/images`,
+     },
+   },
+   `gatsby-transformer-sharp`,
+   `gatsby-plugin-sharp`,
+ ],
+```
+
+Note the `gatsby-source-filesystem` path pointing to the location of images in your project.
+
+2.
+
+```query {
+     file(relativePath: { eq: "gatsby-astronaut.png" }) {
+       childImageSharp {
+         fluid(grayscale: true) {
+           ...GatsbyImageSharpFluid
+         }
+       }
+     }
+   }
+```
+
+Note: The relative path is the image path that you configured in `gatsby-config.js`.
```

Comment:
```suggestion
Note: The relative path is the image path that you configured in `gatsby-config.js`.
```
I think it might be more helpful to specifically call out `gatsby-source-filesystem`? It's possible they'll have other paths configured for other plugins.